<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ENGRENAGES_DEPORTEES</class>
 <widget class="QWidget" name="ENGRENAGES_DEPORTEES">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ENGRENAGES DEPORTEES</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>reducteur_gearexpert.gif</normaloff>reducteur_gearexpert.gif</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>874</width>
        <height>369</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_2_CH20">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>24</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>DETERMINATION DES DEPORTS
 SANS VARIATION D'ENTRAXE</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton_CH20">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>24</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>DETERMINATION DES DEPORTS
 AVEC VARIATION D'ENTRAXE</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_4_CH20">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>24</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>ENGRENAGE DEPORTE
 SANS VARIATION D'ENTRAXE</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_3_CH20">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>24</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>ENGRENAGE DEPORTE
 AVEC VARIATION D'ENTRAXE</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="pushButton_5_CH20">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>RETOUR</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
